### ISSUE TITLE
Explore False Positives in AXE Color Contrast Checks

### ISSUE THREAD

**Issue by holdenhinkle** [department-of-veterans-affairs/va.gov-team#17246]
## Issue Description
Color contrast false positives sometimes occur which can cause tests to fail.  

---
## Tasks
- [ ] Research what can cause false positives and how to avoid them.
- [ ] Update documentation with any relevant findings.
